src/foam/nanos/column/TableColumnOutputter.js

```diff
@@ -44,7 +44,7 @@ foam.CLASS({
             return ( val / 100 ).toString();
           }
           if ( foam.core.DateTime.isInstance(prop) ) {
-            return val.toString().substring(0, 24);
+            return val.toString();
```

Comment:
Do you need/want timezone info?

Reply:
think we need the timezone to avoid inconsistency for agents among different time zones.
otherwise the same report would have a different time depending on timezones, e.g. mine shows 10am yours would be 13pm.

Reply:
sorry, if that picture was there before. I did not see it.